setup.py

```diff
@@ -23,6 +23,16 @@ def check_output(*popenargs, **kwargs):
 
 import sys
 
+if os.environ.get('COVERAGE','') == 'true':
```

Comment:
Also can you check if COVERAGE is already used elsewhere by the `coverage` module?